### ISSUE TITLE
Printing wallet in black-and-white identifies wallet as Bitcoin, not Bitcoin Cash

### ISSUE THREAD

**Issue by fortran77** [theantnest/bccaddress#27]
When I print wallets in black-and-white (actually greyscale), the green portion of the word "BitcoinCash" disappears, leaving behind only "Bitcoin".

It would be nice to have an option to generate paper wallets in black-and-white, ready for printing, with all graphics intact.

The current "Hide Art" checkbox almost achieves this. But it makes the QR codes tiny and harder to scan reliably. We need another checkbox "Black-and-White" that preserves the art and the larger QR codes but simply make it greyscale instead of in color.

**Comment by zquestz** [comment]
Agreed, there is a PR up already to allow more templates to be submitted. Hopefully someone gives us a nice black & white one. =)

**Comment by fortran77** [comment]
I was able to make a few changes in the CSS to reduce the Hide Art wallets to 5 per page, making the QR codes bigger enough to be easily scanned. But I didn't get all the proportions right, as I have to scale down the print output to 93% or less to make it properly fit the page. My diffs are below.

If somebody could redo the CSS to make the Hide Art wallets properly print 5 per page, that would be a good solution pending new image templates.

Diffs for cashaddress.org.html are below, Hide art wallets 5 per page, requires scaling to 93% or less for proper printing.

    15709c15709
    < #paperkeyarea .keyarea { height: 110px; }
    ---
    > #paperkeyarea .keyarea { height: 165px; }
    15712c15712
    < .keyarea .pubaddress { display: inline-block; height: 40px; padding: 0 0 0 10px; float: left; }
    ---
    > .keyarea .pubaddress { display: inline-block; height: 60px; padding: 0 0 0 10px; float: left; }
    15753c15753
    < #paperarea { min-height: 120px; display: none; }
    ---
    > #paperarea { min-height: 180px; display: none; }
    18977c18977
    <   pageBreakAtDefault: 7,
    ---
    >   pageBreakAtDefault: 5,
    19101c19101
    <           ninja.qrCode.showQrCode(keyValuePair);
    ---
    >           ninja.qrCode.showQrCode(keyValuePair, 3);

**Comment by zquestz** [comment]
Template support has been merged. Would love a submission for a nice looking black and white template!

**Comment by fortran77** [comment]
If somebody with the right expertise could edit the existing template so that the green "Cash" part of "BitcoinCash" is in a darker shade, that remains visible on a black-and-white printout, that would be a nice quick fix.

**Comment by Yibhvat** [comment]
Good all working and happy for earn money...

**Comment by atommicios** [comment]
atomic.ios1@gmail.com

**Comment by Unknown566jpg** [comment]
Where do i find my bitcoin address key

**Comment by qprbn4sks6-oss** [comment]
bc1qs3kuze9wc9gnwm09haffx7aedtppa9s6euwj08
